client/components/follow-button/test/button.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { expect } from 'chai';
+import { render } from 'enzyme';
```

Comment:
Cool. I considered that way, thought `render` much simpler to understand. How big is performance hit? Still worth changing?